<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reader_mainmenu</class>
 <widget class="QWidget" name="reader_mainmenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>读者菜单</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>60</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>10</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>身份    : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="query_reader">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询个人信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="query_book">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询图书信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submit_opinion">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>580</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>意见反馈</string>
   </property>
  </widget>
  <widget class="QPushButton" name="borrow_message">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询借阅信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="query_opinion_result">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>750</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询反馈结果</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_button">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>260</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_type">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>10</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string> reader</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_name">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>60</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>cc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_password">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>180</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
